public/js/actions/sources.js

```diff
@@ -240,16 +231,17 @@ function loadSourceText(source) {
       // It's already loaded or is loading
       return Promise.resolve(textInfo);
     }
-    const generatedSource = getGeneratedSource(getState(), source);
-    const originalSources = getOriginalSources(getState(), generatedSource)
-                            .map(s => s.toJS());
```

Comment:
This was setup so that we could show "loading" messages in the original source mapped locations, but it made the action async and was adding marginal benefits. 

Now, we just show "loading" in the generated source, which is fine